ga4gh/client.py

```diff
@@ -685,11 +686,11 @@ def _run_get_request(self, object_name, protocol_response_class, id_):
             response.text, protocol_response_class)
 
     def _run_list_reference_bases_page_request(self, id_, request):
```

Comment:
It appears the existing machinery hardcodes the search ending, and so I believe you'd be introducing machinery to handle the URL to the other functions, or leave them functionally isolated as they are. I'm happy to approach it either way.